OpenStack-Rabbit-Consumer/rabbit-consumer-chart/readme.md

```diff
@@ -1,3 +1,38 @@
 - Install secret from an existing krb5.keytab:
 
-`kubectl create secret generic rabbit-consumer-keytab --from-file krb5.keytab`
\ No newline at end of file
+`kubectl create secret generic rabbit-consumer-keytab --from-file krb5.keytab`
+
+- Install secrets for the Rabbit and Openstack credentials
+  based on the following .yaml template:
+
+```
+kind: Namespace
+apiVersion: v1
+metadata:
+  name: rabbit-consumer
+  labels:
+    name: rabbit-consumer
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  # This should match the values.yaml values
+  name: openstack-credentials
+  namespace: rabbit-consumer  
+type: Opaque
+data:
+  OPENSTACK_USERNAME:
+  OPENSTACK_PASSWORD:
```

Comment:
Should we add a placeholder for the credentials here e.g. `OPENSTACK_USERNAME: <USERNAME> , OPENSTACK_PASSWORD: <PASSWORD>` ?

Reply:
`kubectl` will set them to those placeholders instead of erroring (with the blank fields), what's your thoughts?

Reply:
Ahh I see
I think we don't necessarily need to put the placeholder in the `values.yaml` file then if `kubectl` sets them to those placeholders